.github/workflows/ci.yml

```diff
@@ -104,7 +104,10 @@ jobs:
         uses: actions-rust-lang/setup-rust-toolchain@v1.4.4
 
       - name: Install Nushell
-        run: cargo install --locked --path=. --profile ci --no-default-features
+        run: cargo install --path . --locked --no-default-features
```

Comment:
In an effort to not repeat this problem in the future, would you mind writing up some comments as to why we removed --profile ci? like, it causes stack overflows when running scripts with nu -c, or something like that.

Reply:
oh yeah for sure :+1:

Reply:
should be good in c0f363aacf7a494b9d954d41b02dafb49c5cfc76